src/KintoID.sol

```diff
@@ -94,6 +94,13 @@ contract KintoID is
     // This function is called by the proxy contract when the implementation is upgraded
     function _authorizeUpgrade(address newImplementation) internal override onlyRole(UPGRADER_ROLE) {}
 
+    function _increaseBalance(address account, uint128 value)
```

Comment:
This one needs to be overriden since `_beforeTokenTransfer` does not do this anymore (more [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md?plain=1#L351))

Reply:
So, this is what the changelog says:
```
- `ERC721`: The internal function `_beforeTokenTransfer` no longer updates balances, which it previously did when `batchSize` was greater than 1. This change has no consequence unless a custom ERC721 extension is explicitly invoking `_beforeTokenTransfer`. Balance updates in extensions must now be done explicitly using `__unsafe_increaseBalance`, with a name that indicates that there is an invariant that has to be manually verified.

```

Note the following: "This change has no consequence unless a **custom ERC721 extension is explicitly invoking** `_beforeTokenTransfer`". As far as I can see, even though when we override `_beforeTokenTransfer` we are kind of explicitly invoking `super._beforeTokenTransfer`, we do that with the same params we received. `batchSize` param is called internally on ERC721Upgradeable.sol with value '1'.

Does this make sense?